cmd/imagedigestexporter/main.go

```diff
@@ -65,5 +65,18 @@ func main() {
 	if err != nil {
 		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
 	}
-	fmt.Println(string(imagesJSON))
+	log.Printf("Image digest exporter output: %s ", string(imagesJSON))
+	f, err := os.OpenFile("/workspace/builder/termination-log", os.O_WRONLY|os.O_CREATE, 0666)
```

Comment:
Shouldn't it be `/builder/…` here ?
I am thinking of the case where a resource would be cloned in `/workspace/builder` (a git resource with a target path)

Reply:
yes, makes sense, should not use `/workspace`, will fix that